### PR TITLE
Add feedback for pension-type-tool

### PR DESCRIPTION
**Pull request by dwhenry**
* Refactor BookingFeedbackForm to FeedbackForm
* Add govspeak renderer

### CODE REVIEW

File: app/forms/feedback_form.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class FeedbackForm
+  include ActiveModel::Model
+
+  attr_accessor :name, :email, :message, :booking_type
+
+  validates :name, presence: true
+  validates :email, presence: true, format: { with: /.+@.+\..+/ }
+  validates :message, presence: true
+  validates :booking_type, inclusion: { in: %w(online_booking pension_type_tool) }
```
Comment (benbarnett) [inline app/forms/feedback_form.rb]: Would consider renaming `booking_type` to something else – not sure the pension type tool is that kind of thing.

Not quite sure what it should be called though!

File: app/controllers/booking_requests_controller.rb [hunk 2]
```diff
@@ -5,17 +5,17 @@ class BookingRequestsController < ApplicationController
   before_action :set_breadcrumbs
 
   def step_one
-    @booking_feedback = BookingFeedbackForm.new
+    @feedback = FeedbackForm.new(booking_type: 'online_booking')
```
Comment (benlovell) [inline app/controllers/booking_requests_controller.rb]: Is there any mileage in defaulting this argument to 'online_booking' so we don't have to specify it everywhere? It's overwhelmingly the normal value passed. Or perhaps a specialised method that initialises the object with that `booking_type`. Perhaps `FeedbackForm.for_online_booking` or something?

File: app/forms/feedback_form.rb (hunk 1 above)
Comment (benlovell) [inline app/forms/feedback_form.rb]: Agreed - perhaps `subject`?

Comment (dwhenry) [inline app/forms/feedback_form.rb]: sorry that was supposed to be `feedback_type`

Comment (benlovell) [inline app/forms/feedback_form.rb]: Makes sense to me.

**Comment by dwhenry** [comment]
@benbarnett @benlovell done and done..

**Pull request merged**